src/components/encoding-pane/filter-shelf.tsx

```diff
@@ -93,6 +200,36 @@ class FilterShelfBase extends React.Component<FilterShelfProps, {}> {
       </span>
     );
   }
+
+  private renderTimeUnitChanger(filter: RangeFilter | OneOfFilter, index: number, domain: number[]) {
```

Comment:
Why don't we re-use `FunctionPicker` so we can have less code and ensure consistent style?